.github/workflows/main.yml

```diff
@@ -1,5 +1,13 @@
 name: Test
 
+#
+# Test strategy
+# 
+# - unit-test:        Run all tests on latest macOS and Xcode.
+# - unit-test-linux:  Run tests on Linux (Swift 5.1) - Not include the Combine extension tests.
+# - integration-test: Run integration test by Xcode 11.3.1 and Swift 5.1.3 with CocoaPods.
```

Comment:
This strategy is not covered run all unit tests on Xcode 11.3.1 and Swift 5.1.3.
This is a trade-off currently.